docs/develop/developers_guide.rst

```diff
@@ -0,0 +1,88 @@
+Developer's Guide
+=================
+
+Description of Branches
+-----------------------
+
+There are several primary repository branches
+for specific development tracks.
+
+main
+____
+This is the stable release branch.
+It is updated from the *develop* branch
+and tagged prior to each release.
+The melodies-monet conda package is created from *main*.
+
+develop
```

Comment:
Also, we had talked about trying to just do `main` branch ("develop") + tags ("stable").

Reply:
It looks like I can't change the branch name of main. @bbakernoaa is that something you have the permissions to do? If so I can update the docs to remove any references to main and just have it describe the develop branch and use of additional tags.

Reply:
Sorry I didn't mean to suggest changing the name. I meant that `main` branch represents the "develop" state and tags represent "stable" state.

Reply:
That makes more sense. I'll update to reflect that.